bson/json_util.py

```diff
@@ -499,4 +587,19 @@ def default(obj, json_options=DEFAULT_JSON_OPTIONS):
             return {"$uuid": obj.hex}
     if isinstance(obj, Decimal128):
         return {"$numberDecimal": str(obj)}
+    if isinstance(obj, bool):
```

Comment:
This is here to prevent booleans from matching the following case `isinstance(obj, integer_types`:

```python
>>> isinstance(True, int)
True
>>> issubclass(bool, int)
True
```